### PR TITLE
Feature/43556 ps report data sync read from dedicated queue

### PR DESCRIPTION
**Pull request by jon-shipley**
Move the sync-results-to-sql function to the new throttled-functions app.

Notes

* functions-app was already using SQL_FUNCTIONS_APP_USER and SQL_FUNCTIONS_APP_PASSWORD credentials
* tslib functions apps - both consumption and throttled functions now use the same credentials.

### CODE REVIEW

**Review by GuyHarwood — COMMENTED** [review]
LGSF

File: func-throttled/host.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "version": "2.0",
+  "functionTimeout": "06:01:23",
```
[the remark this answers is not in the document]
Comment (jon-shipley) [reply func-throttled/host.json:3]: It's likely to be remembered and disambiguates between system timeout's and user-set timeout values.

File: functions-app/config.js [hunk 2]
```diff
@@ -37,9 +37,9 @@ module.exports = {
     Encrypt: getEnvWithTypeOrDefault('SQL_ENCRYPT', cast.toBoolean, true, process.env),
     EnableArithAbort: getEnvWithTypeOrDefault('SQL_ENABLE_ARITH_ABORT', cast.toBoolean, true, process.env),
     Application: {
-      Name: process.env.SQL_APP_NAME || 'functions-app', // docker default
-      Username: process.env.SQL_FUNCTIONS_APP_USER || 'functionsAppSystemUser', // docker default
-      Password: process.env.SQL_FUNCTIONS_APP_USER_PASSWORD || 'functionsAppSystemP4ssw0rd!' // docker default
+      Name: process.env.SQL_APP_NAME || 'mtc-functions-app', // docker default
```
[the remark this answers is not in the document]
Comment (jon-shipley) [reply functions-app/config.js:40]: Not much. But it's not harmful either.

File: func-throttled/service-bus-injector.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  await queueClient.close()
+  await busClient.close()
+}
+
+// let's get the party started
+main()
+  .then(() => {
+    console.log('all done')
+  })
+  .catch(error => { console.error(error) })
```
Comment (GuyHarwood) [inline func-throttled/service-bus-injector.js:47]: new line

**Review by GuyHarwood — APPROVED** [review]
LGTM, just a new line to sort

File: func-throttled/service-bus-injector.js (hunk 3 above)
Comment (jon-shipley) [reply func-throttled/service-bus-injector.js]: added, thanks!

**Pull request merged**